compiler/rustc_parse/src/lexer/mod.rs

```diff
@@ -131,7 +131,7 @@ impl<'a> StringReader<'a> {
     ) -> DiagnosticBuilder<'a, !> {
         self.sess
             .span_diagnostic
-            .struct_span_fatal(self.mk_sp(from_pos, to_pos), &format!("{}: {}", m, escaped_char(c)))
+            .struct_span_fatal(self.mk_sp(from_pos, to_pos), &format!("{m}: {}", escaped_char(c)))
```

Comment:
Should be 
```rust
.struct_span_fatal(self.mk_sp(from_pos, to_pos), &format!("`{m}`: {}", escaped_char(c)))
```
?

Reply:
Should that also apply to line 146 of that file ?

Reply:
oh, sorry, I mis-guide you.
No need to add `...

But, I'm not sure whether we should change to this style, because there are too many places use the current style.

Reply:
This style does improve readability in most cases and the new style usage is dotted around the different files, so I though I would help to unify it to the new style.

Reply:
Ehm, does anyone know why CI is failing, I am very new to this rust repo ?

Reply:
The CI is failing, seems the code format is not right.
Please run ` ./x.py test tidy --bless` before pushing to remote.

FYI: https://rustc-dev-guide.rust-lang.org/building/suggested.html